README.md

```diff
@@ -45,9 +45,9 @@ Check the [compatibility matrix][compatibility-matrix] for additional informatio
 
 ### Prerequisites
 
-| Tool                    | Version   | Description                                                                                                                                                |
-| ----------------------- | --------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| [furyctl][furyctl-repo] | `>=0.25.0` | The recommended tool to download and manage KFD modules and their packages. To learn more about `furyctl` read the [official documentation][furyctl-repo]. |
+| Tool                    | Version    | Description                                                                                                                                                |
+| ----------------------- |------------| ---------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [furyctl][furyctl-repo] | `>=0.27.0` | The recommended tool to download and manage KFD modules and their packages. To learn more about `furyctl` read the [official documentation][furyctl-repo]. |
```

Comment:
shouldn't this be 0.27.2?
```suggestion
| [furyctl][furyctl-repo] | `>=0.27.2` | The recommended tool to download and manage KFD modules and their packages. To learn more about `furyctl` read the [official documentation][furyctl-repo]. |
```

Reply:
I don't know if last patch is required. Maybe it will work also with 0.27.0

Reply:
you are right 👍  the containerd features don't need the features included in furyctl's latest patch